ecs-agent/api/resource/ebs_discovery_windows.go

```diff
@@ -0,0 +1,70 @@
+//go:build windows
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package resource
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	log "github.com/cihub/seelog"
+	"github.com/pkg/errors"
+)
+
+const (
+	volumeDiscoveryTimeoutDuration = 5 * time.Second
+)
+
+func ConfirmEBSVolumeIsAttached(ctx context.Context, deviceName string, volumeID string) error {
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, volumeDiscoveryTimeoutDuration)
+	defer cancel()
+	commandArguments := []string{"diskdrive", "where", fmt.Sprintf("SerialNumber like '%%%s%%'", volumeID), "get", "SerialNumber"}
+	output, err := exec.CommandContext(ctxWithTimeout, "wmic", commandArguments...).CombinedOutput()
```

Comment:
Q: Can we also get the device name along with the serial number with `wmic`?

Reply:
We cannot. It's not returned by the command.